### PR TITLE
ZK-4684: menuitem icon alignment

### PR DESCRIPTION
**Pull request by plijyfes**

### CODE REVIEW

File: zktest/src/archive/test2/B95-ZK-4684.zul [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+		test in iceblue and iceblue_c (and tablet styles)
+		click " File" to open menu
+		the save icon and "Save" label should be vertical align
+	</label>
+	<menubar width="100%">
+		<menuitem label="Menu"/>
+		<menu label="File">
+			<menupopup>
+				<menuitem label="View"/>
+				<menuitem iconSclass="z-icon-save" label="Save"/>
```
Comment (plijyfes) [inline zktest/src/archive/test2/B95-ZK-4684.zul:26]: test img

**Review by scribetw — CHANGES_REQUESTED** [review]
test image + iconmSclass

**Review by DevChu — APPROVED** [review]

**Review by scribetw — APPROVED** [review]

**Comment by scribetw** [comment]
@zkadmin rebase

**Comment by zkadmin** [comment]
I just pushed up the changes, enjoy!

**Pull request merged**